test-apps/appui-test-app/standalone/eslint.config.js

```diff
@@ -0,0 +1,44 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+const prettierConfig = require("eslint-config-prettier");
+
+const customLanguageOptions = {
+  sourceType: "module",
+  parser: require("@typescript-eslint/parser"),
+  parserOptions: {
+    project: ["./tsconfig.backend.json", "./tsconfig.json"],
+    ecmaVersion: "latest",
+    ecmaFeatures: {
+      jsx: true,
+      modules: true,
+    },
+  },
+};
+
+module.exports = [
+  {
```

Comment:
youll want to make sure to add an entry for the uiconfig for src/frontend/**

Reply:
this is applicable to all test apps, i just didnt want to repeat myself and make multiple comments

Reply:
I see what you mean but the older config does not have the uiconfig included. Are you suggesting that I should make this addition despite the older version not having these? 
![image](https://github.com/iTwin/appui/assets/98850418/fe1e5322-1550-4a5c-aae1-1a577450ac64)

Reply:
The ones importing in `eslint.config.base.js` should have that config btw